src/ooxml/java/org/apache/poi/xddf/usermodel/chart/XDDFChart.java

```diff
@@ -712,10 +734,29 @@ public CellReference setSheetTitle(String title, int column) {
         XSSFRow row = this.getRow(sheet, 0);
         XSSFCell cell = this.getCell(row, column);
         cell.setCellValue(title);
-        this.updateSheetTable(sheet.getTables().get(0).getCTTable(), title, column);
+        
+        CTTable ctTable = this.getSheetTable(sheet);
+        
+        this.updateSheetTable(ctTable, title, column);
         return new CellReference(sheet.getSheetName(), 0, column, true, true);
     }
 
+    /**
```

Comment:
method which check whether sheet have table object, in case table size is 0 then create table and return that table.